tools/editor/property_editor.cpp

```diff
@@ -3855,6 +3855,25 @@ void PropertyEditor::update_tree() {
 	}
 }
 
+void PropertyEditor::_draw_transparency(Object *t, const Rect2& p_rect) {
+
+	TreeItem *ti=t->cast_to<TreeItem>();
+	if (!ti)
+		   return;
+
+	Color color=obj->get(ti->get_metadata(1));
+	Ref<Texture> arrow=tree->get_icon("select_arrow");
+
+	// make a little space between modulate and self-modulate
```

Comment:
Comment might be nice to change, e.g. `// make a little space between consecutive color fields`